<ui version="4.0">
 <class>Vokabeltrainer</class>
 <widget name="Vokabeltrainer" class="QWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vokabekltrainer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="le__call">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_solution">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_solve">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_call">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_solution">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_feedback">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_level">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
